about.html

```diff
@@ -0,0 +1,112 @@
+
+            <p class="storyParagraphAbout2">At Nova Cosmetics, we believe in the power of natural beauty. Our mission is to create high-quality, natural products that enhance your natural beauty and make you feel confident and empowered. <br>We are dedicated to using only the finest, all-natural ingredients in our products. Our team of scientists and skincare
+            experts work tirelessly to research and develop formulations that are not only effective, but also gentle and nourishing <br> for your skin.
+            We are committed to sustainability and ethical practices, and we source our ingredients from reputable, environmentally<br>
+            responsible suppliers. We never test on animals and all of our products are vegan and cruelty-free.
+            </p>
+            <div id="aboutContact">
+            <p>
+                Please contact us through our email address If you had any other questions or inquiries <br>
+                about Nova Cosmetics
+            </p>
+            <a href="demo@novact.com">demo@novact.com</a>
+            </div>
+        </article>
+        <article class="textAbout2">
+            <h2>Nova Cosmetics 2022</h2>
+            <p>
+                The logo of Nova Cosmetics was decided through the logo competition from 8, June to 7, July.
+            </p>
+            <div class="logoAbout">
+                <img class="logoAbout" src="./Assets/images/logo.png" alt="">
+            </div>
+        </article>
+    </section>
+    <section class="secSectionAbout">
+        <article>
+            <h2>See the past Nova Events</h2>
+            <hr>
+            <p>Take a look at the past Nova events which took place in Dubai and in Erbil</p>
+        </article>
+        <section class="imgTextabt">
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/event22.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/Event_Photo.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        </section>
+        <article class="scroll partners">
+            <h2>Partners</h2>
+            <hr>
+            <div class="partnerLogos">
+            <img class="logoImg" src="./Assets/images/loreal.png" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/mlogo.png" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/Sephora-Logo.png" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/MAC-Cosmetics-Logo.png" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/maynyc.png" alt="Partner Logo">
+            </div>
+        </article>
+    </section>
+    <footer>
+        <p>Nova Logo</p>
+        <p>2022 Nova Cosmetics, Some rights reserved</p>
```

Comment:
**OPTIONAL** - Please stick to the design template for the footer, the background color is different from what is in the design template: Please fix this 🙏, also remember to include the image of your logo in the footer 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="955" alt="wkmae;ktmhse" src="https://user-images.githubusercontent.com/73607512/210562237-be7e8db4-2ed8-42ee-bc0a-aa24865b77b4.PNG">
Figma template:
<img width="554" alt="bknwrknwr" src="https://user-images.githubusercontent.com/73607512/210562321-9f38df25-7e1e-4208-a92b-c06af4f72b47.PNG">